homeassistant/helpers/entity.py

```diff
@@ -281,7 +281,7 @@ def state_attributes(self) -> dict[str, Any] | None:
         return None
 
     @property
-    def device_state_attributes(self) -> Mapping[str, Any] | None:
+    def device_state_attributes(self) -> MutableMapping[str, Any] | None:
```

Comment:
I think we should keep return values of these methods immutable unless this is needed somewhere.
```suggestion
    def device_state_attributes(self) ->Mapping[str, Any] | None:
```